README.md

```diff
@@ -14,13 +14,21 @@ Cordova 5.0.0+ is fully supported, along with the following asociated platforms:
 - Android ([cordova-android](https://github.com/apache/cordova-android) 4.0.0+)
 - iOS ([cordova-ios](https://github.com/apache/cordova-ios) 3.9.0+)
 
-To check which versions of each Cordova platform you are currently using, you can run the following command:
+To check which versions of each Cordova platform you are currently using, you can run the following command and inspect the `Installed platforms` list:
 
 ```shell
-cordova platform version
+cordova platform
+```
+
+If you're running an older Android and/or iOS platform than is mentioned above, and would be open to upgrading, you can easily do so by running the following commands (omitting a platform if it isn't neccessary):
+
+```shell
+cordova platform upgrade android
```

Comment:
What is the minimum version of the CLI supporting this? I just tried it on 5.4.0 and it doesn't seem to work.